changelog/12280.bugfix.md

```diff
@@ -0,0 +1 @@
+Addresses Regular Expression Denial of Service vulnerability in slack connector
```

Comment:
We should link to the CVE link if possible for more details.

Reply:
The link to synk isn't public but I've added a link for RegEx Denial of Service